scripts/bench/runner.js

```diff
@@ -41,7 +41,7 @@ async function runBenchmarks(reactPath) {
 
     if (
       !benchmarkFilter ||
-      (benchmarkFilter && benchmarkName.indexOf(benchmarkFilter) !== -1)
+      (benchmarkName.indexOf(benchmarkFilter) !== -1)
```

Comment:
Should we use:
```js
if (!benchmarkFilter || benchmarkName.includes(benchmarkFilter))
```
Will it be more idiomatic and readable in JavaScript?

Reply:
Good catch!